src/ts/editor/ui/parts/preview/toolbar.ts

```diff
@@ -58,6 +60,9 @@ export class PreviewToolbarPart extends BasePart implements UiPartComponent {
             break;
           }
         }
+      } else if (devices.length >= 1) {
+        // Default to the first device in the list if none selected.
+        this.device = devices[0];
```

Comment:
Suggestion: Let's default to the last device since it's most likely to be the "desktop" view. The desktop view should be the default for most cases.

Reply:
No 'mobile first' design?

Reply:
Realistically in Blinkk we get desktop designs first and we generally build out the desktop designs simultaneously with or before the mobile designs. I can't think of a project where I've received mobile designs before desktop designs, despite the best practice of designing for mobile first.

Reply:
Hmm.. I think instead I'll add a `default` setting to the device config so people can manually specify which device is the default. That way if they have an large desktop it does not become the default since it is last.

Reply:
That sounds fine. I think most of our projects won't configure these settings, and will just rely on the defaults. I'd like the default preview to be the 1400x900 preview (currently "Desktop") for when nothing's selected -- that makes the most sense for our projects and how work is conducted.